hera_cal/io.py

```diff
@@ -877,9 +898,37 @@ def update_vis(infilename, outfilename, filetype_in='miriad', filetype_out='miri
         raise TypeError("Input filetype must be either 'miriad', 'uvfits', or 'uvh5'.")
 
 
+def to_HERACal(input_cal):
+    '''Converts a string path, UVData, or HERAData object, or a list of any one of those, to a
```

Comment:
Fix doc string. I think this should say "Converts a string path, UVCal, or HERACal..."